router/router_test.go

```diff
@@ -332,12 +336,30 @@ func (s *IntegrationSuite) TestSendMessage() {
 	json.NewDecoder(res.Body).Decode(token)
 	assert.Equal(s.T(), "backup-server", token.Name)
 
+	req = s.newRequest("POST", "client", `{"name": "mock-client"}`)
```

Comment:
This feels a little out of place here, the test only tests the message api and now it also checks clients. I think it'd be enough to check if last seen was updated in the authentication package, and that the proposed GetActiveClientTokens method works correctly one of the stream tests.

Reply:
I agree, this was done to make coverage happy,